bin/pre-install.js

```diff
@@ -88,7 +88,7 @@ function checkYarn(agents /*: AgentsVersion */) {
 
   if (agents.yarn) {
     const version = agents.yarn;
-    if (versionCompare(version, '1.0') < 0) {
+    if (versionCompare(version, '1.1') < 0) {
```

Comment:
This is semver versioning and not a floating number, therefore 1.1 != 1.10. It goes like this: 1.9 -> 1.10 -> 1.11. We should use `'1.10'` here.